www/docs/mp/index.php

```diff
@@ -245,7 +245,11 @@
 if (count($current_offices) > 0) {
     $position = $current_offices[0];
 } elseif (count($former_offices) > 0) {
-    $position = $former_offices[0];
+    # only use the position if it's current, i.e.
+    # it does not have an end date
+    if (!$former_offices[0]->to_date) {
```

Comment:
I don't think this line will ever match, as all the former_offices have end dates from `if ($include_only == 'previous' AND $to_date == '9999-12-31')` line (in fact, even current offices will have `9999-12-31`, looks like). Think we either don't show a former position at all, or stick "Former" in front of it?